### ISSUE TITLE
UnrecognizedClientException: The security token included in the request is invalid

### ISSUE THREAD

**Issue by kkumarkarthik** [AllocZero/EfficientDynamoDb#167]
Hello,

We followed the following link to utilize .NET SDK AWSCredentials  

https://alloczero.github.io/EfficientDynamoDb/docs/dev-guide/sdk-compatibility

Following is the piece of code we used 

**Amazon.Runtime.AWSCredentials awsCredentials = options.Credentials;
new DynamoDbContext(new DynamoDbContextConfig(AWSConfig.REGION, awsCredentials.ToCredentialsProvider()));**

We are getting the following error when try to query

**UnrecognizedClientException: The security token included in the request is invalid.**

Alternatively, we tried using EfficientDynamoDb.Configs.AwsCredentials  class and it worked with the same awsCredentials object.

**Amazon.Runtime.AWSCredentials awsCredentials = options.Credentials;
new DynamoDbContext(new DynamoDbContextConfig(AWSConfig.REGION, new AwsCredentials(awsCredentials.GetCredentials().AccessKey, awsCredentials.GetCredentials().SecretKey)))**

Could you please assist with this?

Thanks
KK

**Comment by garibtig** [comment]
@kkumarkarthik I can confirm that this soluction work for me
Can you share with code ? 

This is our wat to use EDDB
``` var endpoint = CreateRegionEndpoint(endpointUrl);
                var credentialsProvider = CreateCredentialsProvider();

                var awsSdkCredentials = new EnvironmentVariablesAWSCredentials();

                var config =
                    new DynamoDbContextConfig(endpoint, env.IsProduction() ? awsSdkCredentials.ToCredentialsProvider() : credentialsProvider)
                    {
                        TableNamePrefix = dbSection[TableNamePrefixName]
                    };

                return config;

**Comment by lezzi** [comment]
Hi @kkumarkarthik, there was a problem related to the credentials token usage. It has been addressed in the `0.9.3` version.

**Comment by kkumarkarthik** [comment]
Thanks, @lezzi for your prompt response and it works with 0.9.3 version
